e2e/key/key.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package key
+
+import (
+	"testing"
+
+	"github.com/kelseyhightower/envconfig"
+	"github.com/sylabs/singularity/e2e/internal/e2e"
+	"github.com/sylabs/singularity/e2e/key/keyprivate"
+	"github.com/sylabs/singularity/e2e/key/keypublic"
+	"github.com/sylabs/singularity/internal/pkg/test"
+)
+
+type testingEnv struct {
+	// base env for running tests
+	CmdPath     string `split_words:"true"`
+	TestDir     string `split_words:"true"`
+	RunDisabled bool   `default:"false"`
+}
+
+var testenv testingEnv
+
+func testGeneralKeyCmd(t *testing.T) {
+	tests := []struct {
+		name    string
+		args    []string
+		file    string
+		succeed bool
+	}{
+		{
+			name:    "key_list",
```

Comment:
you omit the underscore. When running with -v that will read more naturally.

Reply:
Okay, I'll remove the `_` :+1: